ManatiCR/retopais
# RP-2: Que es reto pais page.

https://manati.atlassian.net/browse/RP-2

How-to-test:
- Aquifer build
- Install site
- Ensure all terms from problems taxonomy have images
- Go to http://retopais.dev/admin/config/retopais/que-es-reto-pais and configure necessary stuff
- Go to http://retopais.dev/que-es-reto-pais, it should look like the prototype (https://www.estudiomanati.com/prototipos/retopais/#p=que_es_reto_pais)

Missing stuff: 
- Right sidebar (separate ticket)
- Correct layout (missing necessary stuff)
